wallet/test/unit_testing/services/repository/repository_online_test.dart

```diff
@@ -0,0 +1,219 @@
+  });
+
+  test("test for get address based on username ", () async {
+    final response = await repository.getAddressBasedOnUsername(MOCK_USERNAME);
+    expect(true, response.isRight());
+    response.fold((l) => throw MOCK_SOMETHING_WENT_WRONG, (r) => expect(MOCK_ADDRESS, r));
+  });
+
+  test("test for get list of balances", () async {
+    final response = await repository.getBalance(MOCK_ADDRESS);
+    expect(true, response.isRight());
+    response.fold((l) => throw MOCK_SOMETHING_WENT_WRONG, (r) => expect(MOCK_BALANCE, r));
+  });
+
+  test("test for createDynamicLinkForRecipeNftShare", () async {
+    final response = await repository.createDynamicLinkForRecipeNftShare(address: MOCK_ADDRESS, nft: MOCK_NFT);
+    expect(true, response.isRight());
+    response.fold((l) => throw MOCK_SOMETHING_WENT_WRONG, (r) => expect(MOCK_DYNAMIC_LINK, r));
+  });
+
+  test("test for get executions for recipe", () async {
+    final response = await repository.getExecutionsByRecipeId(cookBookId: MOCK_COOKBOOK_ID, recipeId: MOCK_RECIPE_ID);
+    expect(true, response.isRight());
+    response.fold((l) => throw MOCK_SOMETHING_WENT_WRONG, (r) => expect(MOCK_EXECUTION_LIST_BY_RECIPE_RESPONSE, r));
+  });
+  test("test for get executions for recipe", () async {
+    final response = await repository.getExecutionsByRecipeId(cookBookId: MOCK_COOKBOOK_ID, recipeId: MOCK_RECIPE_ID);
+    expect(true, response.isRight());
+    response.fold((l) => throw MOCK_SOMETHING_WENT_WRONG, (r) => expect(MOCK_EXECUTION_LIST_BY_RECIPE_RESPONSE, r));
+  });
+
+  test("test for get item", () async {
+    final response = await repository.getItem(cookBookId: MOCK_COOKBOOK_ID, itemId: MOCK_ITEM_ID);
+    expect(true, response.isRight());
+    response.fold((l) => throw MOCK_SOMETHING_WENT_WRONG, (r) => expect(MOCK_ITEM, r));
+  });
+
+  test("test for get list item by owner", () async {
+    final response = await repository.getListItemByOwner(owner: MOCK_USERNAME);
+    expect(true, response.isRight());
+    response.fold((l) => throw MOCK_SOMETHING_WENT_WRONG, (r) => expect([MOCK_ITEM], r));
+  });
+
+  test("test for get execution based on id", () async {
+    final response = await repository.getExecutionBasedOnId(id: MOCK_ID);
+    expect(true, response.isRight());
+    response.fold((l) => throw MOCK_SOMETHING_WENT_WRONG, (r) => expect(MOCK_EXECUTION, r));
+  });
+
+  test("test for get trade based on creator", () async {
+    final response = await repository.getTradesBasedOnCreator(creator: MOCK_USERNAME);
+    expect(true, response.isRight());
+    response.fold((l) => throw MOCK_SOMETHING_WENT_WRONG, (r) => expect([MOCK_TRADE], r));
+  });
+
+  test("test for getStripeAccountExistsFromLocal", () async {
+    final response = repository.getStripeAccountExistsFromLocal();
+    expect(true, response.isRight());
+    response.fold((l) => throw MOCK_SOMETHING_WENT_WRONG, (r) => expect(MOCK_STRIPE_ACCOUNT_EXISTS, r));
+  });
+
+  //TODO: HAVE TO REFACTOR THESE CASES LATER
```

Comment:
Kindly do it in this PR>

Reply:
Yes, Added those now.
Initially, we were unable to get the responses from Stripe.